roles/adoption_osp_deploy/tasks/prepare_undercloud.yml

```diff
@@ -54,6 +54,17 @@
         script: "{{ _container_prepare_cmd }}"
       when: cifmw_adoption_osp_deploy_scenario.container_prepare_params is not defined
 
+    # Adoption requires Ceph 7 (Reef) as a requirement. Instead of performing a Ceph
+    # upgrade from 6 (the default) to 7, let's try to deploy 7 in greenfield
+    - name: Patch containers-prepare-parameters Ceph container images
+      when: cifmw_adoption_osp_deploy_scenario.container_prepare_params is not defined
```

Comment:
@cescgina do you think I need this `when` condition or I can patch this file regardless of the variable?

Reply:
you do not need the `when`, as the file should be defined regardless. However, I just realized that I forgot to implement that task for the path when that when does not apply, I'll post a quick PR for that

Reply:
https://github.com/openstack-k8s-operators/ci-framework/pull/2474

Reply:
Thank you, makes sense. Also do you know how we can test this patch? (e.g. a testproject where we replace install_yamls w/ ci-framework driven deploy?)

Reply:
yes, I can add it to my testproject

Reply:
Thank you!